tests/signature_verification.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2022 Alibaba Cloud
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! Test for signature verification.
+
+use std::process::Command;
+
+use image_rs::image::ImageClient;
+
+mod common;
+
+/// Name of different signing schemes.
+const SIMPLE_SIGNING: &str = "Simple Signing";
+const NONE_SIGNING: &str = "None";
+
+/// Script for preparing Simple Signing GPG signature file.
+const SIGNATURE_SCRIPT: &str = "scripts/install_test_signatures.sh";
+
+/// Decrypt config
+const DECRYPT_CONFIG: &str = "provider:attestation-agent:sample_kbc::null";
```

Comment:
The name `DECRYPT_CONFIG` is not clear here. Maybe you can use AA_PARAMETERS or other.

Reply:
Sure, it will seem clearer